src/operator/tensor/ordering_op-inl.h

```diff
@@ -455,8 +457,7 @@ void TopKImpl(const RunContext &ctx,
   // Cast `ret_indices` from int to real_t could introduce conversion error when the element_num
   // is large enough.
   if (param.ret_typ == topk_enum::kReturnMask) {
-    Tensor<xpu, 2, DType> ret_mask =
-      ret[0].get_with_shape<xpu, 2, DType>(Shape2(ret[0].Size(), 1), s);
+    Tensor<xpu, 1, DType> ret_mask = ret[0].FlatTo1D<xpu, DType>(s);
     ret_mask = scalar<DType>(0);
```

Comment:
Now it raises a really weird "CUDA Misaligned Memory Error". I currently having no idea what triggers it. Actually it happens when we initialize the ret_mask to all zero.

Reply:
@anirudh2290 @azai91 @apeforest @samskalicky @eric-haibin-lin - maybe one of you guys can help?

Reply:
@sxjscience were you able to resolve the error?